recipe/meta.yaml

```diff
@@ -32,8 +32,10 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - curl  # [win]
+    - python   3.6  # [osx]
```

Comment:
Interesting that `osx` needs 3.6

Reply:
I'm not entirely sure it's needed, but it's hard for me to debug in CI.

Reply:
If it is working don't touch it :smile: